Core/Src/main.c

```diff
@@ -56,6 +67,9 @@ void SystemClock_Config(void);
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
 
+const uint8_t MCP3004_START = 0b00000001;
+const uint8_t MCP3004_SINGLEENDED_CH0 = 0b10000000; // 2nd MSB and 4 LSBs are don't cares
```

Comment:
Although not necessary, it would be best to use the HEX numbering system to represent these number because it is much easier to read 2 hex numbers instead of 8 binary numbers... just think about if you had to configure a 32 bit register... using binary would be impracticable so it is good practice to use hex!